### ISSUE TITLE
Tree view items are now only clickable by the text itself in CLion 2022.2

### ISSUE THREAD

**Issue by LoganDark** [ChrisRM/material-theme-jetbrains#1879]
### Summary

In the filesystem tree view, you can no longer double-click on the item to open it in the editor. You must double-click on the *text itself*. This happened after updating to CLion 2022.2 from 2021.3

### Expected behavior

Double-click anywhere on it and it opens the file

### Actual behavior

It only opens the file if you double-click directly on the text

### Steps to reproduce (if needed)

1. Open tree view
2. Double-click on something

### Environment

```
CLion 2022.2
Build #CL-222.3345.126, built on July 26, 2022
Licensed to Logan Darklock
Subscription is active until May 5, 2023.
For educational use only.
Runtime version: 17.0.3+7-b469.32 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 10 10.0
GC: G1 Young Generation, G1 Old Generation
Memory: 12288M
Cores: 12
Registry:
    run.processes.with.pty=TRUE
    ide.balloon.shadow.size=0
    cidr.max.intellisense.file.length=1500000

Non-Bundled Plugins:
    org.rust.lang (0.4.176.4779-222-nightly)
    com.samvtran.plugins.macosforallkeymap (2.0.0)
    com.chrisrm.idea.MaterialThemeUI (7.7.0)
    WGSL (0.0.17)
    String Manipulation (9.5.0)
    com.mallowigi (65.1.0)
    GLSL (1.21)
```

[material_theme.xml.txt](https://github.com/ChrisRM/material-theme-jetbrains/files/9227876/material_theme.xml.txt)

**Comment by mallowigi** [comment]
Fixed in 7.7.1. And please use the material-theme-issues to report issues, this repository is archived

**Comment by LoganDark** [comment]
> Fixed in 7.7.1. And please use the material-theme-issues to report issues, this repository is archived

I knew there must've been a reason for this having so few issues open... sorry
